multisig/simulations/ethereum/vip-063/index.ts

```diff
@@ -0,0 +1,38 @@
+import { expect } from "chai";
+import { Contract } from "ethers";
+import { ethers } from "hardhat";
+import { checkIsolatedPoolsComptrollers } from "src/vip-framework/checks/checkIsolatedPoolsComptrollers";
+
+import { forking, pretendExecutingVip } from "../../../../src/vip-framework";
+import { NEW_VTOKEN_IMPLEMENTATION, VTOKEN_BEACON, vip063 } from "../../../proposals/ethereum/vip-063";
+import VTOKEN_BEACON_ABI from "./abis/upgradableBeacon.json";
+
+forking(20862480, async () => {
+  let vTokenBeacon: Contract;
+
+  before(async () => {
+    vTokenBeacon = new ethers.Contract(VTOKEN_BEACON, VTOKEN_BEACON_ABI, ethers.provider);
+  });
+
+  describe("Pre-VIP behavior", () => {
+    it("check implementation", async () => {
+      expect(await vTokenBeacon.implementation()).to.be.equal("0xE5A008B6A0bAB405343B3ABe8895966EAaFb5790");
+    });
+  });
+
+  describe("Post-VIP behavior", async () => {
```

Comment:
Can we also check based on time and the number of blocks for additional validation purpose?